container/devel/Dockerfile.openqa.leap

```diff
@@ -0,0 +1,34 @@
+FROM registry.opensuse.org/opensuse/leap:15.6
+
+RUN zypper ar -G https://download.opensuse.org/repositories/devel:/openQA/15.6 devel-openqa \
+        && zypper ar -G https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.6/15.6/ devel-openqa-leap \
```

Comment:
I suggest to use priorities as explained in http://open.qa/docs/#_development_version_repository